src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -186,6 +186,11 @@ table tbody ul {
     padding: 0 0 0 1em;
 }
 
+table thead th.num-col,
+table tbody td.num-col {
+    text-align: center;
```

Comment:
numeric values are usually aligned right

Reply:
No strong opinion on that, this is how it look for `right` align:
![render-count-right-align](https://user-images.githubusercontent.com/2028198/42822550-51614554-89a9-11e8-8dce-fe35bea44a68.png)
If someone else has the same criteria, I'll change it with pleasure.